packages/shared/src/components/profile/ProfileSettingsMenu.tsx

```diff
@@ -87,32 +87,38 @@ const useMenuItems = (): NavItemProps[] => {
       { label: 'Edit profile', icon: <EditIcon />, href: '/account/profile' },
     ];
 
-    items.push({
-      label: isPlus ? 'Manage plus' : plusCta,
-      icon: <DevPlusIcon />,
-      href: plusHref,
-      className: isPlus ? undefined : 'text-action-plus-default',
-      target:
-        isPlus && plusProvider === SubscriptionProvider.Paddle
-          ? '_blank'
-          : undefined,
-      onClick: () => {
-        if (
-          isPlus &&
-          isIOSNative() &&
-          plusProvider === SubscriptionProvider.AppleStoreKit
-        ) {
-          sendMessage(WebKitMessageHandlers.IAPSubscriptionManage, null);
-        }
+    if (
+      !isIOSNative() ||
+      (isIOSNative() && user.isTeamMember)
```

Comment:
Just to make sure we don't show the link before we're ready and have tested it in production.